client/src/pages/Terms&Condition/TermsConditions.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function TermsConditions() {
+  return <div>
+HELLO FROM TERMS & cONDITIONS
```

Comment:
can you please add space here also (cONDITIONS) looks need to be all uppercase

Reply:
It's just to a placeholder. will delete later